### PR TITLE
Qdrant client version roll up

### PR DESCRIPTION
**Pull request by gaya3-vijayakumar**
## What

Qdrant client version roll up

## Why

...

## How

...

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions / Env Variables

-

## Notes on Testing

Unable to generate pdm. lock with this as for some reason, pin resolution is looping forever with boto3. Tried alternates like pinning down boto3. But no luck. Since pdm.lock for adapters is only used by developers, @chandrasekharan-zipstack and @gaya3-zipstack have decided to skip this.

## Screenshots

image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -27,7 +27,8 @@ dependencies = [
         "vecs==0.4.3",
         "pinecone-client==2.2.4",
         "psycopg2-binary==2.9.9",
-        "qdrant-client==1.7.0",
+        #Any change
```
Comment (chandrasekharan-zipstack) [inline pyproject.toml]:
```suggestion
```
@gaya3-zipstack what is this comment conveying? I think it can be removed

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM

File: pyproject.toml (hunk 1 above)
Comment (gaya3-vijayakumar) [reply pyproject.toml]: I removed it.

Comment (gaya3-vijayakumar) [reply pyproject.toml]: Thanks!

**Review by jaseemjaskp — APPROVED** [review]

**Pull request merged**